rep-0003.rst

```diff
@@ -180,6 +179,48 @@ Buildsystem support:
 
   - build from source
 
+Kinetic Kame (May 2016 - May 2021)
+----------------------------------
+Required Support for:
+
+- Ubuntu Wily (15.10)
+- Ubuntu X (16.04)
+
+Minimum Requirements:
+
+- C++03
+
+  - Some C++11 will be enabled (TODO(esteve): need to determine which ones)
```

Comment:
Making c++11 the minimum requirement would allow the use of `std::shared_ptr` but In general I would recommend that we not plan to change APIs from things like `boost::shared_ptr` to `std::shared_ptr` or even make them overloaded. The only case would be if we specifically review those APIs and want to run them through a tick tock cycle. However it is virtually infeasible to drop boost as a dependency for ROS 1.0 thus changing the APIs away from the boost APIs will simply incur costs of changes without significant benefits of reducing the dependency. And it will effect a large amount of code.